src/markdown-pages/docs/ecosystem/ecosystem-architecture.md

```diff
@@ -0,0 +1,68 @@
+
+<br>Expand the following sections to find out more about the key components of the Galasa ecosystem:
+
+<details>
+<summary>Galasa Stores</summary>
+
+|                                   |                                                                                                                          |
+| ------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------- |
+| **Configuration Property Store**                       | The configuration property store (CPS) defines object properties, topologies, system configurations, and definitions which instruct the way in which a Galasa test runs. For example, properties for endpoints, ports and timeouts. When running in an ecosystem, all Galasa tests will use the same CPS configuration, unless any overrides are passed at submission. It is the CPS and the configurational properties that enable tests to run against multiple environments, without changing the code inside the test.         |
+| **Dynamic Status Store**               | The dynamic status store (DSS) provides status information about the ecosystem and the tests that are running. The DSS is used by the resource manager and engine controller to ensure the limits that are set in the CPS configuration are not exceeded. DSS property values change dynamically as tests are run, showing the resources that are currently being used, shared or locked by a test, so that workloads can be limited to avoid throttling. When running in automation, the DSS is shared by every instance of the framework.                                                             |
+| **Result Archive Store**                      | The result archive store (RAS) is a single database which stores all elements of a test, including the test results, run logs, and test artifacts. These elements can be used to help diagnose the cause of any failures encountered as a result of running a test, or to gather information about a test.  Storing all of this information in one place makes it simple for entire teams to view results.                              |
+| **Credentials Store**                   | The credentials store (CREDs) securely provides the credentials, for example, password, username and authentication token that are required for a test to run in automation. The CREDs is hosted in the etcd server.    |
+| **etcd**                       | The etcd server is a highly available key-value pair store which hosts the Configuration Property Store (CPS), the Dynamic Status Store (DSS) and the Credentials Store (CREDs). The CPS, DSS contain information that are used by all tests and the CREDs is used only for automation runs.  The etcd server stores and maintains a single, consistent source of the truth about the status of the ecosystem at any given point in time.      |
```

Comment:
I think we need to clarify what is mean by the CPS and DSS contain info used by all tests and CREDS is only used for automation runs. We could even probably just drop the sentence